stories/components/ValueSetter.test.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint react/jsx-props-no-spreading: off */
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { ValueSetter } from './ValueSetter';
+import '@testing-library/jest-dom/extend-expect';
+
+const propsSample = {
+  initialValue: 20000,
+  children: (val: number) => {
+    return val;
+  },
+  upLabel: 'increase',
+  downLabel: 'decrease',
+};
+
+describe('<ValueSetter component functionality testing/>', () => {
```

Comment:
This component really only exists to make writing Storybook stories easier, I don't think we need to test it.